setup.cfg

```diff
@@ -18,5 +18,46 @@ formats = zip
 universal = 1
 
 [metadata]
-license_file = LICENSE
+name = setuptools
 version = 41.0.1
+description = Easily download, build, install, upgrade, and uninstall Python packages
+author = Python Packaging Authority
+author_email = distutils-sig@python.org
+long_description = file: README.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+license_file = LICENSE
+keywords = CPAN PyPI distutils eggs package management
+url = https://github.com/pypa/setuptools
+project_urls =
+    Documentation = https://setuptools.readthedocs.io/
```

Comment:
May I suggests adding more sidebar links to the PyPI page? (CIs, docs etc.)
Examples:
- https://github.com/sanitizers/octomachinery/blob/f056cda/setup.cfg#L11-L15
- https://github.com/cherrypy/cheroot/blob/247e6a2/setup.cfg#L10-L16